pkg/config/remote/uptane/util.go

```diff
@@ -65,6 +67,11 @@ func metaVersion(rawMeta json.RawMessage) (uint64, error) {
 	return *metaVersion.Signed.Version, nil
 }
 
+func metaHash(rawMeta json.RawMessage) string {
+	hash := sha256.Sum256(rawMeta)
+	return hex.EncodeToString(hash[:])
```

Comment:
How many helper functions like this probably exist in the world? Probably anybody that does anything with hashes has this since they don't give us something in the stdlib! 😆

Reply:
Maybe with go template it will be better